deployment/ansible/roles/raster-foundry.worker/tasks/configuration.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Configure application
+  copy: content="{{ item.value }}"
+        dest="{{ envdir_home }}/{{ item.key }}"
+        owner=root
+        group=rf
+        mode=0750
+  with_dict: worker_config
```

Comment:
If this is identical to `app_config`, and templating these variables goes into `base`, then this whole task might be able to go.